applications/HDF5Application/python_scripts/hdf5_io.py

```diff
@@ -283,3 +283,31 @@ def _get_current_file_name(self):
         fmt = "{:." + str(self._time_tag_precision) + "f}"
         time_tag = "-" + fmt.format(self._model_part.ProcessInfo[KratosMultiphysics.TIME])
         return self._time_step_file_name + time_tag + ".h5"
+
+class InitializationFromInputProcess(KratosMultiphysics.Process):
+    """A process to initialize a simulation using an HDF5 output file."""
```

Comment:
I think this is like a static version of TemporalInputProcess, but it behaves slightly differently in how you parse the file_name. Can we adopt the same behaviour for parsing the file_name and call it something like StaticInputProcess or InitialInputProcess to be slightly more general?

Reply:
What I mean is to do

```
default_settings = KratosMultiphysics.Parameters("""{
        {
                 "file_name": "%s"	                 "file_name": "%s"
        }
    }""" % model_part.Name)
```